UnitTests/HopsanCoreTests/ComponentUtilitiesTest/tst_componentutilitiestesttest.cpp

```diff
@@ -538,6 +538,83 @@ private Q_SLOTS:
         QTest::newRow("4") << tempVec << 0.001 << 1.0 << -1000000000.0 << 1000000000.0 << (10000.0)*0.001;
         QTest::newRow("5") << tempVec << 0.001 << 1.0 << -1000000000.0 << 2.0 << 2.0;
     }
+
+    void ploParser()
+    {
+        QFETCH( QString, ploData);
+        QFETCH( QString, indexDataName);
+        QFETCH( QString, outputDataName);
+        QFETCH( bool, expectParseOK);
+        QFETCH( int, expectedInIndex);
+        QFETCH( int, expectedOutIndex);
+        QFETCH( double, expectedLastValue);
+
+        PLOParser parser;
+        bool parsedOK = parser.readText(qPrintable(ploData));
+        QCOMPARE(parsedOK, expectParseOK);
+        int iIn = parser.getColIdxForDataName(qPrintable(indexDataName));
+        int iOut = parser.getColIdxForDataName(qPrintable(outputDataName));
+        QCOMPARE(iIn, expectedInIndex);
+        QCOMPARE(iOut, expectedOutIndex);
+        if ((iIn >= 0) && (iOut >= 0)) {
+            std::vector<double> values;
+            bool copyOK = parser.copyColumn(iOut, values);
+            QVERIFY(copyOK);
+            QCOMPARE(values.back(), expectedLastValue);
+        }
+    }
+
+    void ploParser_data()
+    {
+        QTest::addColumn< QString >("ploData");
+        QTest::addColumn< QString >("indexDataName");
+        QTest::addColumn< QString >("outputDataName");
+        QTest::addColumn< bool >("expectParseOK");
+        QTest::addColumn< int >("expectedInIndex");
+        QTest::addColumn< int >("expectedOutIndex");
+        QTest::addColumn< double >("expectedLastValue");
+
+        QString ploData1 = R"(
+          'VERSION'
+          1
```

Comment:
Raw string literals only works for C++11 and above (but I guess you noticed  this already).